api/tests/unit/domain/usecases/sessions-mass-import/validate-sessions_test.js

```diff
@@ -82,6 +96,9 @@ describe('Unit | UseCase | sessions-mass-import | validate-sessions', function (
         sessionRepository,
         sessionCodeService,
         i18n,
+
+        sessionsImportValidationService,
+        temporarySessionsStorageForMassImportService,
```

Comment:
Pinaillage, un saut de ligne inutile ^^ (sur les suivants aussi)

```suggestion
        sessionsImportValidationService,
        temporarySessionsStorageForMassImportService,
```

Reply:
Merci ! Corrigé